starboard/android/shared/test_filters.py

```diff
@@ -72,6 +72,12 @@
         # when invalid initialization data is passed to
         # SbDrmGenerateSessionUpdateRequest().
         'SbDrmSessionTest.InvalidSessionUpdateRequestParams',
+
+        # This test crashed after NDK 25 upgrade.
```

Comment:
Please try to keep tracking bugs for disabled tests in the comment lines here. With our current setup it's easier to manage the follow-ups and cleanups if they are logged as bugs.

I know this file doesn't have them, but we have tried for other platforms ( e.g. go/cobalt-cl/252900 )